packages/react-app-revamp/tailwind.config.js

```diff
@@ -57,11 +57,6 @@ const remToPx = rem => {
 
 module.exports = {
   content: ["./components/**/*.{js,jsx,ts,tsx}", "./pages/**/*.{js,jsx,ts,tsx}", "./layouts/**/*.{js,jsx,ts,tsx}"],
-  purge: {
```

Comment:
so when I was working on landing, I had to add `purge` in tailwind.config in order to load these animations.

however, I just initialized your branch and it looks like it's working without it! so lgtm, could be due to our upgrades that it is not needed anymore.

Reply:
Oh sweet, alright awesome!!